File: 백준/Bronze/2562. 최댓값/최댓값.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace MyCompiler {
    class Program {
        public static void Main(string[] args) {

            // pair 값 초기화
            (int first, int second) pair = (0, 0);
            
            for(int i = 1; i <= 9; i++)
            {
                int num = int.Parse(Console.ReadLine());

                if(num > pair.first)
                {
                    pair.first = num;
                    pair.second = i;
                }
            }

            Console.WriteLine($"{pair.first}\n{pair.second}");
        }
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <bits/stdc++.h>

using namespace std;

int main() {

    // 값과 인덱스 저장할 pair
    pair<int, int> maxValue;

    maxValue.first = 0;
    maxValue.second = 0;

    // 9개의 숫자 입력 받기
    for(int i = 1; i <= 9; i++)
    {
        int num = 0;
        cin >> num;
        
        if(num > maxValue.first)
        {
            maxValue.first = num;
            maxValue.second = i;
        }
    }

    // 값과 인덱스 출력
    cout << maxValue.first << '\n' << maxValue.second;
}